narwhals/series.py

```diff
@@ -440,6 +440,42 @@ def std(self, *, ddof: int = 1) -> Any:
         return self._series.std(ddof=ddof)
 
     def is_in(self, other: Any) -> Self:
+        """
+        Check if elements of this sequence are present in the other Series.
+
```

Comment:
We are checking if the Series is in the `other` but `other` is not a Series. 

So I think it should be:
Check if the elements of this Series are in the `other` sequence.

Reply:
Thanks @DeaMariaLeon for the comment, I corrected the summary